h/api/storage.py

```diff
@@ -140,3 +146,7 @@ def _prepare(annotation):
     transform.normalize_annotation_target_uris(annotation)
 
     transform.add_nipsa(annotation)
+
+
+def _postgres_enabled():
+    return pyramid.threadlocal.get_current_request().feature('postgres')
```

Comment:
@seanh we now pass the `request` into the storage functions because we also want to query database specifically from the passed-in DB session :+1: